.github/workflows/ci.yml

```
@@ -16,14 +16,24 @@ jobs:
     env:
       CC: gcc-14
       CXX: g++-14
+      SRC_DIR: ${{ github.workspace }}/src
+      TEST_DIR: ${{ github.workspace }}/tests
+      TEST_ASSETS_ROOT: /tmp/pingpong_tracker
+      SCRIPTS_DIR:  ${{ github.workspace }}/scripts
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)